bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ChannelStateDescriptionProvider.java

```diff
@@ -57,7 +61,29 @@ public StateDescription getStateDescription(String itemName, Locale locale) {
             Channel channel = thingRegistry.getChannel(channelUID);
             if (channel != null) {
                 ChannelType channelType = thingTypeRegistry.getChannelType(channel, locale);
-                return channelType != null ? channelType.getState() : null;
+                if (channelType != null) {
+                    StateDescription state = channelType.getState();
```

Comment:
Could you call the variable `stateDescription`? (I know that the name of the getter method of ChannelType is not ideal...)